libs/labelbox/src/labelbox/schema/ontology.py

```diff
@@ -95,6 +101,13 @@ class Type(Enum):
     classifications: List[Classification] = field(default_factory=list)
     schema_id: Optional[str] = None
     feature_schema_id: Optional[str] = None
+    attributes: Optional[FeatureSchemaAttributes] = None
+
+    def __post_init__(self):
+        if self.attributes is not None:
+            warnings.warn(
+                "Attributes are an experimental feature and may change in the future."
```

Comment:
can you use a std message as described here (for beta) https://labelbox.atlassian.net/wiki/spaces/ENG/pages/3178037250/SDK+Experimental+Features